### ISSUE TITLE
Redundant role 'button'

### ISSUE THREAD

**Issue by benperiton** [carbon-design-system/carbon-components-svelte#988]
I've just fired up a project I haven't touched in a couple of months, and now seem to be getting Ally warnings:

```
WARNING in ./node_modules/carbon-components-svelte/src/Dropdown/Dropdown.svelte
A11y: Redundant role 'button' (201:6)
201:       role="button"
```

```
WARNING in ./node_modules/carbon-components-svelte/src/UIShell/GlobalHeader/Header.svelte
A11y: Redundant role 'banner' (81:8)
81: <header role="banner" aria-label="{ariaLabel}" class:bx--header="{true}">
```

Not entirely sure what has changed, but I see the it was removed from Button before (https://github.com/carbon-design-system/carbon-components-svelte/pull/559)

**Comment by metonym** [comment]
What version of Svelte are you using?

**Comment by metonym** [comment]
Fixed in [v0.51.2](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.51.2)
